### PR TITLE
add some icons to template selection dialog

### PR DESCRIPTION
**Pull request by octaeder**
Add icons for builtin and internet categories.
I find it quite hard to follow the change of radio buttons at the lower end of the window while selecting templates. I created an icon that can tell that this is a multifile template. While those automatically choose the directory option, now for non multifile templates it is the editor option (I find this quite natural).

image

Guess where the multifiles are.

### CODE REVIEW

File: src/templateselector.cpp [hunk 1]
```diff
@@ -498,6 +502,7 @@ void TemplateSelector::showInfo(QTreeWidgetItem *currentItem, QTreeWidgetItem *p
 			ui.rbCreateInFolder->setChecked(true);
 			ui.rbCreateInEditor->setEnabled(false);
 		} else {
+			ui.rbCreateInEditor->setChecked(true);
```
Comment (sunderme) [inline src/templateselector.cpp:505]: This is probably not a good choice, as the user may want to set it in a folder. It should not be changed if the user switched to folder.

**Comment by Alexander-Wilms** [comment]
Maybe Colibre already contains the required icons?

File: src/templateselector.cpp (hunk 1 above)
Comment (octaeder) [reply src/templateselector.cpp:505]: ok, with the multifile icon it should be clear that a switch happened.

**Comment by octaeder** [comment]
> Maybe [Colibre](https://github.com/rizmut/libreoffice-style-colibre) already contains the required icons?

I think you use icon set Colibre, but I use Classic. So I should set icon names depending on the selected icon set, right?

**Comment by sunderme** [comment]
> > Maybe [Colibre](https://github.com/rizmut/libreoffice-style-colibre) already contains the required icons?
> 
> I think you use icon set Colibre, but I use Classic. So I should set icon names depending on the selected icon set, right?

no, you should use a general name which is used in the different icon sets (on linux from the system theme).
getIcon searches for that icon and falls back to built-in if not present

**Comment by sunderme** [comment]
folder-cloud or folder-html  may be your "internet"

**Comment by Alexander-Wilms** [comment]
I've created a PR against your fork: https://github.com/octaeder/texstudio/pull/1

Colibre:

image

Oxygen Modern:

image

**Comment by octaeder** [comment]
thanks @Alexander-Wilms 

Now we have:
image

**Comment by sunderme** [comment]
thanks
